src/value-converters/abstract-locale-value-converter.ts

```diff
@@ -32,14 +31,13 @@ export abstract class AbstractLocaleValueConverter {
     /**
      * Called when the instance or system locale has changed
      */
-    protected localeChanged(locale:string) {
+    protected localeChanged(_locale: string) {
```

Comment:
Why renaming it to `_locale`?

Reply:
Many local variables have been renamed.

Reply:
Because it's unused. See @Zsar's comment [further up](https://github.com/T-Systems-MMS/aurelia-components/pull/62#discussion_r970782381). Without the underscore, the compiler complains about it being unused, with the underscore it's okay.

Reply:
I don't have a strong opinion about that, because I don't care about unused parameters. I keep them there for documentation reasons and readability.